builder/hyperv/common/driver.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
```

Comment:
I am fine with any license that is required.  I did not choose any license explicitly.